scripts/platforms/README_v2.md

```
@@ -0,0 +1,179 @@
+## Quick Start
+
+### 1. Prerequisites
+
+Make sure you have:
+- `applicant.json` — Your personal info (name, email, phone, etc.)
+- `candidate_profile.txt` — Your professional summary/experience
+- `job_description.txt` — The job posting text
+- `resumes/generated/tailored_resume.pdf` — Your resume
+
+And environment variables set:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)